.travis.yml

```diff
@@ -21,22 +21,23 @@ before_script:
   - cd $TRAVIS_BUILD_DIR
 script:
   # Fetch dependencies.
-  - go get -t -v ./...
-  - GO111MODULE=on go get -v ./...
+  #- go get -d -t -v ./...
+  - GO111MODULE=on go get -d -v ./...
 
   # Consult Go fmt to ensure consistent code style.
   - diff -u <(echo -n) <(gofmt -d -s .)
 
   # Consult golangci-lint (multiple Go linting tools).
-  - golangci-lint run . ./elem/... ./event/... ./example/...
-  - golangci-lint run --exclude 'exported .* should have comment .*or be unexported' ./prop/... ./style/... # https://github.com/gopherjs/vecty/issues/227
-  - GOOS=js GOARCH=wasm golangci-lint run --build-tags 'js wasm' . ./elem/... ./event/... ./example/...
-  - GOOS=js GOARCH=wasm golangci-lint run --build-tags 'js wasm' --exclude 'exported .* should have comment .*or be unexported' ./prop/... ./style/... # https://github.com/gopherjs/vecty/issues/227
+  #- golangci-lint run . ./elem/... ./event/... ./example/...
+  #- golangci-lint run --exclude 'exported .* should have comment .*or be unexported' ./prop/... ./style/... # https://github.com/gopherjs/vecty/issues/227
+  #- GOOS=js GOARCH=wasm golangci-lint run --build-tags 'js wasm' . ./elem/... ./event/... ./example/...
+  #- GOOS=js GOARCH=wasm golangci-lint run --build-tags 'js wasm' --exclude 'exported .* should have comment .*or be unexported' ./prop/... ./style/... # https://github.com/gopherjs/vecty/issues/227
 
-  # Test with Go compiler and GopherJS compiler.
-  - go test -v -race ./...
-  - gopherjs test -v ./...
+  # Test with Go compiler (under amd64 and wasm architectures) and GopherJS compiler.
+  #- go test -race ./...
+  - GOOS=js GOARCH=wasm go test -exec="$(go env GOROOT)/misc/wasm/go_js_wasm_exec" ./... # https://github.com/golang/go/wiki/WebAssembly#executing-webassembly-with-nodejs
```

Comment:
FYI there is https://github.com/agnivade/wasmbrowsertest which allows you to test against a browser.

Reply:
Neat, I can see why that'd be useful -- though one of our goals with Vecty is to make `go test` alone (even without GOOS=js) work (initially just for testing Vecty itself, secondarily/not in this PR in projects using Vecty)